gulpfile.js

```diff
@@ -120,12 +137,46 @@ function webpackComponents(done) {
 		done();
 	})
 }
+
 function webpackServices(done) {
 	var webpack_config = require('./build/webpack/webpack.services.js')
 	webpack(webpack_config, function (err, stats) {
 		done();
 	})
 }
+
+function angularBuild(done) {
+	try {
+		var baseCmd = 'ng build --base-href "/components/'
+
+		var process = function (row) {
+			var compName = row.source.split("/").pop();
+			var cwd = path.join(__dirname, row.source);
+			var command = baseCmd + compName + '/" --outputPath "' + path.join(__dirname, row.source, row["output-directory"]) + '"';
+			
+			// switch to components folder
+			var dir = shell.pwd();
+			shell.cd(cwd);
+			var outputNpm = shell.exec("npm install"); // CLI doesn't install NPM modules, mmake sure this happens
```

Comment:
We shouldn't do this here

Reply:
Is this build required because the Angular components have their own package.json and the CLI doesn't run install? If that is the case, then it probably would be better for the CLI to output a message with the install command, and have this output an error if the install had not been executed.

Reply:
Probably shouldn't do it here no... however out of the box the component isn't build-able until this has happened. The CLI should be doing it but isn't - I will work on adding it there. For now I can either add an error check and kill the build with a message, or just run it once for them if node_modules doesn't exist.

Reply:
I suspect trying to to integrate the builds (of seed project with React components and Angular components) would be an arse - they do share many common components (inc. Sass and Webpack) but I'd probably avoid trying to produce a build that deals with both React and Angular together. We may have clients turn up with an existing Angular component, highly customized build process and ultimately a need to separate the builds... 

However, we are generating the Angular project in the CLI (using `ng new`), so if we can get a combined build to work, great. However, we also use whatever the current Angular CLI installed is so we would likely have to test compatibility at different versions and all new major Angular milestones.

Thoughts? I can try going the other way on this, but think it might take a while...

Reply:
Removing the `npm install` entirely - the CLI does do it... It was a nasty interaction between having Angular CLI installed in the finsemble project and in the child project. Angular projects can be components of other build envs, but not another Angular project. I had installed the CLI locally initially (due to Angular warning messages) those message *must* be ignored during component generation - hence will put a big health warning in docs (when I write them).